REFERENCE.md

```diff
@@ -8,18 +8,18 @@
 
 #### Public Classes
 
-* [`postgresql::client`](#postgresqlclient): Installs PostgreSQL client software. Set the following parameters if you have a custom version you would like to install.
-* [`postgresql::globals`](#postgresqlglobals): Class for setting cross-class global overrides.
-* [`postgresql::lib::devel`](#postgresqllibdevel): This class installs postgresql development libraries.
-* [`postgresql::lib::docs`](#postgresqllibdocs): Installs PostgreSQL bindings for Postgres-Docs. Set the following parameters if you have a custom version you would like to install.
-* [`postgresql::lib::java`](#postgresqllibjava): This class installs the postgresql jdbc connector.
-* [`postgresql::lib::perl`](#postgresqllibperl): This class installs the perl libs for postgresql.
-* [`postgresql::lib::python`](#postgresqllibpython): This class installs the python libs for postgresql.
-* [`postgresql::server`](#postgresqlserver): This installs a PostgreSQL server
-* [`postgresql::server::contrib`](#postgresqlservercontrib): Install the contrib postgresql packaging.
-* [`postgresql::server::plperl`](#postgresqlserverplperl): This class installs the PL/Perl procedural language for postgresql.
-* [`postgresql::server::plpython`](#postgresqlserverplpython): This class installs the PL/Python procedural language for postgresql.
-* [`postgresql::server::postgis`](#postgresqlserverpostgis): Install the postgis postgresql packaging.
+* [`postgresql::client`](#postgresql--client): Installs PostgreSQL client software. Set the following parameters if you have a custom version you would like to install.
```

Comment:
This will break navigation on Forge, and I think that's the reason where existing one skipped double `hyphen` (--).

Reply:
Here is the tracking ticket for Forge to fix the same : https://perforce.atlassian.net/browse/PF-2807

Reply:
It was already used in https://github.com/puppetlabs/puppetlabs-postgresql/pull/1500 but Simon used an older puppet-strings version by accident to regenerate it.